packages/api/src/external/commonwell/admin/patch-patient-duplicates.ts

```diff
@@ -0,0 +1,74 @@
+import { getPersonId, getPersonIdFromUrl } from "@metriport/commonwell-sdk";
+import { getPatientOrFail } from "../../../command/medical/patient/get-patient";
+import MetriportError from "../../../errors/metriport-error";
+import { Util } from "../../../shared/util";
+import { setCommonwellId } from "../patient-external-data";
+import { getCWAccessForPatient } from "./shared";
+
+/**
+ * IMPORTANT: This is an Admin-only function, it should not be executed outside of internal endpoints.
+ *
+ * Patch duplicated persons in CommonWell - sibling of find-patient-duplicates.ts
+ *
+ * Links the patient to the chosen person.
+ * Additionally, unenroll:
+ * - any other person linked to the patient; AND
+ * - all other persons matching the patient's demographics.
```

Comment:
nit: `all other persons matching the patient's demographics if the unenrollByDemographics flag is set`